FaunaDB.Client/FaunaDB.Client.csproj

```diff
@@ -87,5 +87,7 @@
     <Compile Include="Query\PageHelper.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="Types\BytesV.cs" />
+    <Compile Include="Encoding\Encoder.cs" />
```

Comment:
I was thinking about this again. Maybe we should make this part of `Types` just so people doesn't have to import another namespace. I imagine this will be used way more than the alternative.